internal/services/storage/storage_account_resource_test.go

```diff
@@ -4107,3 +4130,53 @@ resource "azurerm_storage_account" "test" {
 }
 `, data.RandomInteger, data.Locations.Primary, data.RandomString)
 }
+
+func (r StorageAccountResource) isSftpEnabledTrue(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-storage-%d"
+  location = "%s"
+}
+
+resource "azurerm_storage_account" "test" {
+  name                = "unlikely23exst2acct%s"
+  resource_group_name = azurerm_resource_group.test.name
+
+  location                 = azurerm_resource_group.test.location
+  account_kind             = "StorageV2"
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+  is_hns_enabled           = true
+  sftp_enabled             = true
```

Comment:
Do we want consistency in naming here? `is_hns_enabled` and `is_sftp_enabled`?

Reply:
I did wonder about that but I felt the `is_` prefix sounds like a question and Terraform is declarative so should be a statement. 
That said, I wonder if `enable_sftp` would be better as it is in-line with `enable_https_traffic_only`

Reply:
fwiw - We would probably look to fix `is_hns_enabled` to `hns_enabled` to align with convention, likely in a major release, so `sftp_enabled` is correct here.